docs/pages/contributing/documentation/reference.mdx

```diff
@@ -156,20 +156,21 @@ documentation experience to their own environment.
 ### The Var component
 
 To declare a variable, use the `Var` component, as shown in the two `code`
-snippets below:
+snippets below. This example includes an extra space before the tag names of the
```

Comment:
```suggestion
snippets. This example includes an extra space before the tag names of the
```
Nit: Typically, it's bad form to use above/below in favor of using preceding or following

### **Above and below**

Don't use *above* to mean *earlier* or as an adjective preceding a noun (*the above section*) or following a noun (*the code above*). Use a hyperlink, or use *previous*, *preceding*, or *earlier*. Don't use *below* to mean *later* or as an adjective preceding a noun (*the below section*) or following a noun (*the code below*). Use a hyperlink, or use *later* or *the following*. For example: 

- Use the preceding code to display information about the database.
- Use the following code to display information about the database.

Reply:
Just curious, what's the reasoning behind this? I'm not sure "following" is necessarily any clearer than "below".

Your suggestion works, in this case, so I'll include it.